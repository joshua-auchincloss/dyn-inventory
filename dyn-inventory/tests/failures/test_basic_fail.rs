// unknown ExtraParams keyword should fail parsing
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    UnknownKeyRegistry<T: PluginApi> {
        id: u32,
        t: T,
    };
    unknown_kw = Bogus,
);

fn main() {}
