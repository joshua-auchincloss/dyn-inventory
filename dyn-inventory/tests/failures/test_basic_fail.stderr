error: unknown keyword
  --> tests/failures/test_basic_fail.rs:18:5
   |
18 |     unknown_kw = Bogus,
   |     ^^^^^^^^^^
